### ISSUE TITLE
No input file error condition printed multiple times

### ISSUE THREAD

**Issue by aidanheerdegen** [coecms/mppnccombine-fast#35]
Running program with no input files prints error message from each PE:
```
$ mpirun -n 8 /home/502/aph502/code/c/mppnccombine-fast/mppnccombine-fast -o ocean_month.nc
mpirun: INFO: switching to OpenMPI runtime version 3.0.1
ERROR: No input files specified
ERROR: No input files specified
ERROR: No input files specified
ERROR: No input files specified
ERROR: No input files specified
ERROR: No input files specified
ERROR: No input files specified
ERROR: No input files specified
```

**Comment by ScottWales** [comment]
Fixed in beta branch #33

**Comment by aidanheerdegen** [comment]
Confirmed fix
